evap/staff/importers.py

```diff
@@ -271,14 +273,19 @@ def read_one_enrollment(self, data):
 
     def process_course(self, course_data, sheet, row):
         course_id = course_data.name_en
+
         if course_id not in self.courses:
             if course_data.name_de in self.names_de:
                 self.errors.append(_('Sheet "{}", row {}: The German name for course "{}" already exists for another course.').format(sheet, row + 1, course_data.name_en))
             else:
                 self.courses[course_id] = course_data
                 self.names_de.add(course_data.name_de)
         else:
-            if not course_data == self.courses[course_id]:
+            if set(course_data.degree_names) != set(self.courses[course_id].degree_names) and \
```

Comment:
I'd prefer one comparison per line here, binary operators at line start. Idc about line endings (with \ or implicit with parentheses) @karyon, @janno42  what do you think?